allofplos/article_class.py

```diff
@@ -292,6 +289,35 @@ def get_dates(self, string_=False, string_format='%Y-%m-%d', debug=False):
 
         return dates
 
+    def dates_debug(self):
```

Comment:
Is this supposed to be in the final class or was this something you were using in an intermediate sense to debug stuff? This might just need a method name change.

Reply:
I intended for it to be there and used it for Corpus QA, but I recognize it may not be as applicable to other users. Open to alternative name suggestions if it merits inclusion!

Reply:
That's a totally valid use case! can you describe exactly why you need to check this? In what circumstances would this need to trigger a warning/error?

Also, if it's being used for QA it should probably raise an Error of some kind (possibly a [`TypeError](https://docs.python.org/3/library/exceptions.html#TypeError), [`RuntimeError`](https://docs.python.org/3/library/exceptions.html#RuntimeError), or a custom Error that you write separately, [Handling Exceptions](https://docs.python.org/3/tutorial/errors.html#handling-exceptions) does an ok job of explaining how to write custom exceptions this but not best-practices).